### DISCUSSION TITLE
blog/2024/04/04/group_by-summarize-ungroup-animations/

### DISCUSSION THREAD

**Discussion by giscus** [andrewheiss/ath-quarto#95] [Blog comments]
# blog/2024/04/04/group_by-summarize-ungroup-animations/

Visually explore how {dplyr}’s more complex core functions work together to wrangle data

https://www.andrewheiss.com/blog/2024/04/04/group_by-summarize-ungroup-animations/

**Comment by t-emery** [comment]
This is awesome.  I will definitely use this with my students next semester!

**Comment by dlizcano** [comment]
Great! Thanks.

**Comment by jfsalzmann** [comment]
Thanks so much! I noticed that the download links for mutate() and for group_by() |> mutate() are the same...
